wiki/Glossary/Elite_Beat_Agents/en.md

```diff
@@ -0,0 +1,16 @@
+---
+tags:
+  - nintendo games
+  - ds games
+  - eba
+---
+
+# Elite Beat Agents
+
```

Comment:
```suggestion
# Elite Beat Agents

*See also: [osu! Tatake! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")* <!-- this needs an stub to link to -->

```
I'm not sure if linking articles outside of the wiki in a "See also" hatnote is ok, but I'll leave it here anyway; ignore this if I'm wrong. **This link should be replaced with a stub within the wiki that talks about the Ouendan games in the future Not sure if this.**